package/score-worker/src/main/java/io/cloudslang/schema/WorkerBeanDefinitionParser.java

```diff
@@ -74,16 +87,19 @@
 		put(RetryTemplate.class, null);
 		put(SimpleExecutionRunnableFactory.class, null);
 		put(WorkerManagerMBean.class, "io.cloudslang.worker.management.services.WorkerManagerMBean");
+		put(WorkerMetricsMBean.class, "io.cloudslang.worker.monitor.service.WorkerMetricsMBean");
 		put(WorkerRecoveryManagerImpl.class, null);
 		put(ReflectionAdapterImpl.class, null);
-        put(SessionDataHandlerImpl.class, "sessionDataHandler");
+		put(SessionDataHandlerImpl.class, "sessionDataHandler");
 		put(SynchronizationManagerImpl.class, null);
-        put(WorkerConfigurationServiceImpl.class, "workerConfiguration");
-
-        //Monitors
-        put(WorkerExecutionMonitorServiceImpl.class, "workerExecutionMonitorService");
-        put(WorkerMonitorsImpl.class, "workerMonitorsImpl");
-        put(ScheduledWorkerLoadMonitor.class, "scheduledWorkerLoadMonitor");
+		put(WorkerConfigurationServiceImpl.class, "workerConfiguration");
+
+		//Monitors
+		put(WorkerExecutionMonitorServiceImpl.class, "workerExecutionMonitorService");
```

Comment:
Please put all the newly added classes in the bottom

Reply:
Done